components/hackthons/components/Searchbar.js

```diff
@@ -1,151 +1,150 @@
-              </IconButton>
-            </InputAdornment>
-          ),
-          "& .MuiFormLabel-root": {
-            fontSize: "0.8rem",
-          },
-        }}
-        InputLabelProps={{ style: { fontSize: 90 } }}
-      />
-
-      <div className="tw-mt-8 tw-flex tw-flex-wrap tw-gap-4">
-        <Chip
-          variant={selectedTag === "All" ? "default" : "outlined"}
-          // color={selectedTag === "All" ? "primary" : "info"}
-          label="ALL"
-          value="All"
-          onClick={() => handleTagChange("All")}
-          style={{
-            fontSize: window.innerWidth > 768 ? "16px" : "14px",
-            padding: "1rem",
-            backgroundColor: selectedTag[0] === "All" ? "#845ec2" : "#fff",
-            borderColor: selectedTag[0] === "All" ? "#845ec2" : "#845ec2",
-            color: selectedTag[0] === "All" ? "#fff" : "#845ec2",
-          }}
-        />
-
-        {InternshipLabels &&
-          InternshipLabels.map((mylabel) => (
-            <Chip
-              key={mylabel}
-              variant={selectedTag === { mylabel } ? "default" : "outlined"}
-              // color={selectedTag === label ? "primary" : "info"}
-              label={mylabel}
-              onClick={() => handleTagChange(mylabel)}
-              style={{
-                fontSize: window.innerWidth > 768 ? "16px" : "14px",
-                padding: "1rem",
-                backgroundColor: selectedTag.toString().includes(mylabel)
-                  ? "#845ec2"
-                  : "#fff",
-                borderColor: selectedTag.toString().includes(mylabel)
-                  ? "#845ec2"
-                  : "#845ec2",
-                color: selectedTag.toString().includes(mylabel)
-                  ? "#fff"
-                  : "#845ec2",
-              }}
-            />
-          ))}
-        {HackathonLabels &&
-          HackathonLabels.map((mylabel) => (
-            <Chip
-              key={mylabel}
-              variant={selectedTag === { mylabel } ? "default" : "outlined"}
-              // color={selectedTag === label ? "primary" : "info"}
-              label={mylabel}
-              onClick={() => handleTagChange(mylabel)}
-              style={{
-                fontSize: window.innerWidth > 768 ? "16px" : "14px",
-                padding: "1rem",
-                backgroundColor: selectedTag.includes(mylabel.toString())
-                  ? "#845ec2"
-                  : "#fff",
-                borderColor: selectedTag.toString().includes(mylabel)
-                  ? "#845ec2"
-                  : "#845ec2",
-                color: selectedTag.toString().includes(mylabel)
-                  ? "#fff"
-                  : "#845ec2",
-              }}
-            />
-          ))}
-      </div>
-    </form>
-  );
-}
-
-export default SearchBar;
+import React, { useState } from "react";
+
+// import styles from '../styles/Searchbar.module.css'; // Import the CSS file
+
+import Chip from "@mui/material/Chip";

```

Comment:
you are using MUI :)

Reply:
In thi pull request ,i have fixed  search input fieldas mentioned ,later i will fix the chips and make pr for it

Reply:
Chip is part of search bar, please you have to fix ik this pR only.